thirdparty/scripts/build_flatbuffers.sh

```diff
@@ -7,7 +7,7 @@ set -e
 
 TP_DIR=$(cd "$(dirname "${BASH_SOURCE:-$0}")"; pwd)/../
 
-FLATBUFFERS_VERSION=1.7.1
```

Comment:
Wait, are we building flatbuffers twice? (excluding the time that Arrow builds it)

Why do we have this script in addition to the `ExternalProject_Add(flatbuffers_ep` in `Common.cmake`?

Reply:
Yeah, I think we can figure this out when we get rid of the old ray code (and get rid of Common.cmake then).